numpy/linalg/linalg.py

```diff
@@ -1963,6 +1969,11 @@ def lstsq(a, b, rcond=-1):
                                  0, work, lwork, iwork, 0)
         lwork = int(work[0])
         liwork = iwork[0]
+        if any(size == 0 for size in (lwork, liwork)):
+            raise RuntimeError(
+                'LAPACK did not compute workspace sizes: len(work) = {}, len(iwork) = {}'
+                .format(lwork, liwork)
+            )
```

Comment:
It turns out that Accelerate based problems creep up everywhere, I've recently bitten by the same lapack version problem -> Scipy discussion https://github.com/scipy/scipy/pull/6051